src/lib/server/summarize.ts

```diff
@@ -1,8 +1,14 @@
 import { buildPrompt } from "$lib/buildPrompt";
 import { generateFromDefaultEndpoint } from "$lib/server/generateFromDefaultEndpoint";
 import { defaultModel } from "$lib/server/models";
+import {
+	LLM_SUMMERIZATION
+} from "$env/static/private";
 
 export async function summarize(prompt: string) {
+	if (!LLM_SUMMERIZATION) {
+		return prompt;
```

Comment:
```suggestion
		return prompt.split(/\s+/g).slice(0,5).join(' ');
```
Maybe we could split the prompt on white spaces and only take the first 5 words, what do you think ?

Reply:
Sound like even better